crew

```diff
@@ -449,8 +449,8 @@ def unpack (meta)
     when /\.zip$/i
       puts "Unpacking archive using 'unzip', this may take a while..."
       _verbopt = @opt_verbose ? 'v' : 'qq'
-      system "unzip", _verbopt, "-d", "#{extract_dir}", meta[:filename]
-    when /\.(tar(\.(gz|bz2|xz))?|tgz|tbz)$/i
+      system "unzip", '-' + _verbopt, "-d", "#{extract_dir}", meta[:filename]
```

Comment:
Why not just include the dash in `_verbopt`?

Reply:
We also need a version bump in `lib/const.rb`.

Reply:
The dash in `_veropt` would be slightly redundant. Version bumped.

Reply:
That seems fair. Give me a minute...